browser/resources/settings/brave_overrides/settings_menu.ts

```diff
@@ -152,7 +150,8 @@ RegisterStyleOverride(
         }
 
         a:hover, cr-icon:hover {
-          color: #A6ABE9 !important;
+          color: var(--leo-color-icon-interactive) !important;
+          --iron-icon-fill-color: var(--leo-color-icon-interactive) !important;
```

Comment:
nit: variable declarations should come first

```suggestion
          --iron-icon-fill-color: var(--leo-color-icon-interactive) !important;
          color: var(--leo-color-icon-interactive) !important;
```